src/main/java/TestFile.java

```diff
@@ -28,7 +28,7 @@ private static void anotherVariationOfIncorrectComparingToItself() {
 
     private static void onMoreVariationOfIncorrectComparingToItself() {
         int x = 23422;
-        if (x <= x) {
+        if (x >= x) {
```

Comment:
[Checkstyle] WARNING: 'if' have incorrect indentation level 8, expected level should be 4.